dist/angular-localForage.js

```diff
@@ -14,7 +14,11 @@
       return factory(angular, localforage);
     });
   } else if(typeof exports === 'object' || typeof global === 'object') {
-    module.exports = factory(angular, require('localforage')); // Node/Browserify
+    if(typeof module === 'undefined') {
+      global.module.exports = factory(angular, require('localforage')); // NW.js
+    } else {
+      modules.exports = factory(angular, require('localforage')); // Node/Browserify
```

Comment:
Fixed the typo, but I am not sure what you want me to require on the REPL to check if it imports. 

If it means anything I manually added my updated code to my working project using NW.js and the fix works in NW.js and in-browser. 

Just want to make sure I am doing the right thing here :)